third_party/silabs/SiWx917_sdk.gni

```diff
@@ -103,15 +138,52 @@ template("siwx917_sdk") {
       "USE_NVM3=1",
       "RSI_ARM_CM4F",
       "SIWX_917",
-      "BRD4325A",
       "CHIP_9117",
-      "RSI_BLE_ENABLE=1",
+      "BLE_ENABLE=1",
+      "RSI_LITTLE_ENDIAN=1",
       "TINYCRYPT_PRIMITIVES",
       "OPTIMIZE_TINYCRYPT_ASM",
       "__error_t_defined",
       "RSI_SAMPLE_HAL",
+      "ENABLE_IPMU_APIS",
+      "ROM_BYPASS",
+      "SI917",
```

Comment:
Why do we need SI917, CHIP_917 and SIWX_917 ?

Reply:
Removed SI917 from defines